workers/social/lib/social/models/invitation.coffee

```diff
@@ -218,7 +238,12 @@ module.exports = class JInvitation extends jraphical.Module
 
           async.parallel queue, (err, codes) ->
             return callback err  if err
-            return callback()  unless returnCodes
+
+            unless returnCodes
+              data = { type: 'create', invitations: codes }
+
+              return notifyGroupOnInvitationChange groupName, data, callback
```

Comment:
sending invitation codes to whole group can be problematic. can u just send the invitee mails only?